app/experimenter/experiments/migrations/0046_experiment_risk_data_category.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 2.1.7 on 2019-04-18 22:01
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Ah a new risk field, now we gotta start doing those annoying data migrations I was mentioning.  

https://docs.djangoproject.com/en/2.2/topics/migrations/#data-migrations

Basically after we add the field, we have to find all existing experiments that have already passed the 'Sign Off' state (Ready to Ship, Accepted by Normandy, Live, Complete) and set any risks that are `None` to `False`.  Check all the risk fields because there's a couple we've added that we didn't do a data migration for and then we can also close https://github.com/mozilla/experimenter/issues/461

Reply:
will do